### PR TITLE
solution

### PR DESCRIPTION
**Pull request by patryk-babon**
[DEMO LINK](https://patryk-babon.github.io/react_clock/)

### CODE REVIEW

**Review by choeqq — CHANGES_REQUESTED** [review]
please make sure your GH pages demo URL works, try running `npm run deploy`, code looks fine

**Comment by patryk-babon** [comment]
Yeah, I forgot about that

**Review by ogospadzipalina — APPROVED** [review]
looks good to me

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,63 @@
...
+    window.clearInterval(this.timerId);
+  }
 
-      <div className="Clock">
-        <strong className="Clock__name">
-          {clockName}
-        </strong>
+  removeClock = () => {
+    this.setState({ hasClock: true });
+  };
```
Comment (mvjl000) [inline src/App.tsx:42]: Why does removeClock sets hasClock to true? Kinda unintuitive 😸

**Review by mvjl000 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (patryk-babon) [reply src/App.tsx]: Corrected

**Review by choeqq — APPROVED** [review]

**Pull request closed without merging**